types/comtac/Cluey/temperature_input_3.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$id": "https://akenza.io/comtac/cluey/temperature_input_3.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "temperature_input_3",
+  "title": "Temperature Input 3",
+  "properties": {
+    "limit": {
+      "title": "Limit",
+      "description": "Limit",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "event": {
+      "title": "Event",
+      "description": "Event",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "interrogation": {
+      "title": "Interrogation",
+      "description": "Interrogation",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "cyclic": {
+      "title": "Cyclic",
+      "description": "Cyclic",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+
+    "invalid": {
+      "title": "Invalid",
+      "description": "Invalid",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "overflow": {
+      "title": "Overflow",
+      "description": "Overflow",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "limit1": {
+      "title": "Limit 1",
+      "description": "Limit 1",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "limit2": {
+      "title": "Limit 2",
+      "description": "Limit 2",
+      "type": "boolean",
+      "hideFromKpis": true
+    },
+    "value": {
+      "title": "Value",
```

Comment:
Also here, do we have a more telling name?

Reply:
I wanted to be consistent with the other schemas, so i did go here for value as well, even tough its most likely gonna be temperature.